modules/gdscript/tests/scripts/parser/features/constants.out

```diff
@@ -0,0 +1,33 @@
+GDTEST_OK
+>> WARNING
+>> Line: 2
+>> UNUSED_LOCAL_CONSTANT
+>> The local constant '_TEST' is declared but never used in the block. If this is intended, prefix it with an underscore: '__TEST'
```

Comment:
This needs a fix in the GDScript code.

Reply:
I don't know how to fix that warning, should I ignore it or should I wait for a fix PR to be made first?

Reply:
I'd say leave like this for now, but open an issue about it.